src/sentry/sentry_metrics/consumers/last_seen_updater.py

```diff
@@ -120,7 +120,12 @@ def __init__(
         self.__prefilter = LastSeenUpdaterMessageFilter(metrics=self.__metrics)
 
     def __should_accept(self, message: Message[KafkaPayload]) -> bool:
-        return not self.__prefilter.should_drop(message)
+        should_accept = not self.__prefilter.should_drop(message)
+
+        if not should_accept:
+            self.__metrics.incr("last_seen_updater.dropped_message")
```

Comment:
we need to make sure that this is sampled because in theory this is an extremely common occurrence (the theory behind this filter is that most messages will only have hardcoded/cache read keys).

Reply:
Forgot to respond here, seems like it is indeed sampled by default